src/backend/web/routes/index.js

```diff
@@ -45,9 +45,15 @@ if (process.env.NODE_ENV === 'development') {
   if (process.env.PROXY_FRONTEND) {
     // Allow proxying the Gatsby dev server through our backend if PROXY_FRONTEND=1 is set in env
     router.use('/', createProxyMiddleware({ target: 'http://localhost:8000', changeOrigin: true }));
-  } else {
     // Or serve the static files in the Gatsby build directory
+  } else if (FRONTEND === 'gatsby') {
     router.use(express.static(path.join(__dirname, '../../../frontend/gatsby/public')));
+  } else if (FRONTEND === 'next') {
+    router.use(express.static(path.join(__dirname, '../../../frontend/next/out')));
+  } else {
+    throw new Error(
```

Comment:
Let's test that this does what we want (e.g., we should probably crash right now).